README.md

```diff
@@ -377,14 +377,15 @@ Extensions are optional additions to the core specification as defined above.  I
 
 ## Custom Projection
 
-There are scenarios where the Reconciler that processes a `ServiceBinding` (hereinafter referred to as `Reconciler A`) is different than the Reconciler that will project the binding into the Application (hereinafter referred to as `Reconciler B`). To transfer the projection responsibility from Reconciler A to Reconciler B the `ServiceBinding` CR author **MUST** set the `projection.service.binding/type` annotation to `Custom`.  
+There are scenarios where the Reconciler that processes a `ServiceBinding` (hereinafter referred to as "Reconciler A") is different than the Reconciler that will project the binding into the Application (hereinafter referred to as "Reconciler B"). To transfer the projection responsibility from Reconciler A to Reconciler B the `ServiceBinding` author **MUST** set the `projection.service.binding/type` annotation to `Custom`.  An exemplar CRD can be found [here][sbp-crd].
 
-Reconciler A reacts to this annotation by creating a new `ServiceBindingProjection` CR which includes the necessary information for Reconciler B, who is watching and takes responsibility for the new `ServiceBindingProjection` resource (see the [Custom Projection Definition](#custom-projection-definition) section below), to carry out the projection.  Reconciler A is responsible for updating `ServiceBindingProjection`'s spec upon corresponding changes to `ServiceBinding`.
+Reconciler A reacts to this annotation by creating a `ServiceBindingProjection` which includes the necessary information for Reconciler B. Reconciler B takes responsibility for the `ServiceBindingProjection` resource to perform the projection.  Reconciler A is responsible for updating the `ServiceBindingProjection`'s `spec` upon corresponding changes to `ServiceBinding`.
 
-Reconciler B **MUST** set the `ServiceBindingProjection`'s `Ready` condition according to the rules set in [Ready Condition Status](#ready-condition-status), while Reconciler A **MUST** reflect `ServiceBindingProjection`'s `Ready` condition in a new `ProjectionReady` condition inside `ServiceBinding`.  The `Ready` condition of `ServiceBinding` **MUST NOT** be set to `True` if its `ProjectionReady` condition is not `True`.
+Reconciler B **MUST** set the `ServiceBindingProjection`'s `Ready` condition according to the rules set in [Ready Condition Status](#ready-condition-status). Reconciler A **MUST** reflect `ServiceBindingProjection`'s `Ready` condition with an additional `ProjectionReady` condition on `ServiceBinding`.  The `Ready` condition of `ServiceBinding` **MUST NOT** be `True` unless `ProjectionReady` is `True`.
```

Comment:
> with an additional `ProjectionReady` condition

this reads a bit funny to me. The additional bit is the condition, which is of type `ProjectionReady`. Two `ProjectionReady` conditions on the same resource would be invalid.

Reply:
This is an interesting scenario.  What if we had a `ServiceBinding` that used labels to define the app, and it matched 2 applications that had different reconcilers?  

I think the spec should restrict the usage to just a single "custom projection" reconciler, so that we never have to create more than 1 `ServiceBindingProjection` per `ServiceBinding` and thus @scothis's assertion that we would only have a single `ProjectionReady` condition is true.

Reply:
> What if we had a `ServiceBinding` that used labels to define the app, and it matched 2 applications that had different reconcilers?

What I would expect to happen based on the current language is that each of the "Reconciler B"s does the work that it can, but since neither is able to complete all of the work, the `ServiceBindingProjection`'s `Ready` condition is never set to `True`. For a less well behaved "Reconciler B"s it could try to update the `Ready` condition because it completed its work creating a potentially false positive `Ready`.

Ultimately, this is a bit of a corner case at the intersection of two power user features. While users will likely be initially confused, the system doesn't fail spectacularly.

> I think the spec should restrict the usage to just a single "custom projection" reconciler, so that we never have to create more than 1 `ServiceBindingProjection` per `ServiceBinding` ...

This is fine for an implementation to implement, but I don't think the spec needs to define how an implementation creates `ServiceBindingProjection`s for a `ServiceBinding` resources. The spec defines the public facing behavior.

> ... and thus @scothis's assertion that we would only have a single `ProjectionReady` condition is true.

That's not quite what I meant. The conditions list traditionally behaves like a set where the `Type` of each condition is a unique key. Having multiple conditions with the same key would be invalid. My suggestion was intended to avoid any confusion there there may be multiple conditions with the same key on the same resource.

Reply:
> What I would expect to happen based on the current language is that each of the "Reconciler B"s does the work that it can, but since neither is able to complete all of the work, the ServiceBindingProjection's Ready condition is never set to True. For a less well behaved "Reconciler B"s it could try to update the Ready condition because it completed its work creating a potentially false positive Ready.

To me this seems very much like the issue we were trying to avoid earlier when we had 2 reconcilers work on the `status` of `ServiceBinding`, which is what ultimately led to the separation into `ServiceBindingProjection`.  I think the spec would be remiss to not take the necessarily precautions to avoid this same situation from happening, just a different resource.

Reply:
@scothis Do you believe that it was my wording change here that changed the meaning of this?  Or do you think that there was always a hole here that is just more prominent?

Reply:
@nebhale "in a new" was potentially ambiguous, "with an additional" increases the ambiguity. But maybe I'm reading too much into it. Within the context of how Conditions work in Kubernetes, the intent is clear in both cases.

Reply:
I'd prefer not to hold this PR on this wording unless you think I've made it worse.  If so, I've got no problem rolling it back and we can have another PR to iterate on it.